index.bs

```diff
@@ -285,8 +284,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>RP ID</dfn>
 :: An identifier for the [RP] on whose behalf a given registration or authentication ceremony is being performed. Scoped
     credentials can only be used for authentication by the same entity (as identified by RP ID) that created and registered
-    them. By default, the RP ID for a WebAuthn operation is set to the [=current settings object=]'s
-    [=origin=]. This default can be overridden by the caller subject to certain restrictions, as specified in
+    them. By default, the RP ID for a WebAuthn operation is set to the [=relevant settings object=] of the {{WebAuthentication}}
+    object's [=origin=]. This default can be overridden by the caller subject to certain restrictions, as specified in
```

Comment:
The RP ID is set to the relevant settings object? Or did you mean the RP ID is set to the origin specified by the WebAuthentication object's relevant settings object?